bigbluebutton-client/src/org/bigbluebutton/modules/whiteboard/views/WhiteboardCanvas.as

```diff
@@ -370,13 +375,19 @@ package org.bigbluebutton.modules.whiteboard.views {
 			if (e.whiteboardId == currentWhiteboardId) {
 				whiteboardToolbar.whiteboardAccessModified(e.multiUser);
 				canvasModel.multiUserChange(e.multiUser);
-				canvasDisplayModel.multiUserChange(e.multiUser);

```

Comment:
Still need to call this function. The canvasDisplayModel.multiUserChange() method should be changes to removing the part that modifies the cursors because they're going to be deleted anyways.